neatnet/artifacts.py

```diff
@@ -747,7 +747,7 @@ def one_remaining(
     es_mask : pandas.Series
         A mask for ``edges`` in the ``E`` and ``S`` continuity groups.
     max_segment_length : float | int
-        Additional vertices will be added so that all line segments
+        Additional nodes will be added so that all line segments
```

Comment:
Well, if we consider node to be linked to the graph representation, we are adding vertices. Not nodes. This change can be confusing and imho is wrong.